### PR TITLE
Make sure astral symbols is counted as a single character

### PR DESCRIPTION
**Pull request by kevva**
Should fix the conflict between full width characters and surrogate pairs.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -67,7 +68,7 @@ module.exports = (str, begin, end) => {
 			++visible;
 		}
 
-		if (isFullwidthCodePoint(x.codePointAt())) {
+		if (isFullwidthCodePoint(x.codePointAt()) && !ASTRAL_REGEX.test(x)) {
```
Comment (kevva) [inline index.js]: Performance wise, testing the regular expression first might be faster.

Comment (sindresorhus) [inline index.js]: Yes, put the regex first.

Comment (kevva) [inline index.js]: Aight, fixed.

**Comment by sindresorhus** [comment]
Awesome! Thanks for fixing all this Kevin :)

**Pull request merged**